execute/execute.py

```diff
@@ -0,0 +1,162 @@
+import argparse
+import json
+import jsonschema
+import time
+
+from pathlib import Path
+
+from console_access_api.aitrios_console import AitriosConsole
+
+parser = argparse.ArgumentParser(
+    description="Launches execution on a camera configured in AITRIOS Console",
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+)
+
+parser.add_argument(
+    "--aitrios_secrets",
+    "-s",
+    type=str,
+    dest="aitrios_secrets",
+    help=f"The path to your AITIRIOS console configuration file",
+    required=True,
+)
+parser.add_argument(
+    "--device_name",
+    "-D",
+    type=str,
+    dest="device_name",
+    help=f"Device Name to search for",
```

Comment:
Unnecessary f-string

```suggestion
    help="Device Name to search for",
```